packages/smooth_app/lib/l10n/app_fr.arb

```diff
@@ -2412,8 +2408,8 @@
     "nova_group_3": "Groupe NOVA 3",
     "nova_group_4": "Groupe NOVA 4",
     "nova_group_unknown": "Groupe NOVA inconnu",
-    "hunger_games_loading_line1": "Please let us a few seconds…",
-    "hunger_games_loading_line2": "We're downloading the questions!",
+    "hunger_games_loading_line1": "Veuillez nous laisser quelques secondes…",
+    "hunger_games_loading_line2": "Nous téléchargeons les questions !",
     "hunger_games_error_label": "Argh! Something went wrong… and we couldn't load the questions.",
-    "hunger_games_error_retry_button": "Let's retry!"
+    "hunger_games_error_retry_button": "Réessayons !"
```

Comment:
In French, I think we can be more formal, "Réessayons" seems weirder than in English